app/code/Magento/SalesRule/Model/ResourceModel/Rule/Customer.php

```diff
@@ -48,4 +48,22 @@ public function loadByCustomerRule($rule, $customerId, $ruleId)
         $rule->setData($data);
         return $this;
     }
+
+    /**
+     * Delete the time Usage from salesrule_customer table when times_used is 0
+     * @param $ruleId
```

Comment:
Please, add param type

Reply:
Hi @sidolov,
 I have added params data type. Kindly check and let me know.